backstage/templates/scaffolder/terraform-control-module/template.yaml

```diff
@@ -193,72 +194,10 @@ spec:
           default: 1.7.4
           description: The version of Terraform to use in the Terraform configuration.
 
-    # todo: ask if we should create the pagerduty integration and if not, ask for the optional key.
-    - title: Integrations (optional)
-      properties:
-        createPagerduty:
-          title: Create PagerDuty Service?
-          description: |
-            Check box to create a new PagerDuty service for this component.
-            If you already have a key, you can input it below.
-            If you don't want to use the PagerDuty integration, leave the field blank.
-          type: boolean
-          default: false
-          # ui:widget: select
-
-      dependencies:
-        createPagerduty:
-          oneOf:
-            - properties:
-                createPagerduty:
-                  enum:
-                    - true
-
-                escalationPolicyId:
-                  title: Escalation Policy ID
-                  type: string
-                  description: The ID of the escalation policy to associate with the service
-                  ui:field: SelectFieldFromApi
-                  ui:options:
-                    title: PagerDuty Escalation Policy
-                    description: Select an escalation policy from PagerDuty
-                    path: 'pagerduty/escalation_policies'
-                    labelSelector: 'label'
-                    valueSelector: 'value'
-                    placeholder: '---'
-                alertGrouping:
-                  title: Alert Grouping
-                  type: string
-                  description: Reduce noise by grouping similar alerts - Defaults to 'None'.
-                  enum:
-                    - 'time'
-                    - 'intelligent'
-                    - 'content_based'
-                  enumNames:
-                    - 'Time-based grouping'
-                    - 'Intelligent grouping'
-                    - 'Content-based grouping'
-
-
-            - properties:
-                createPagerduty:
-                  enum:
-                    - false
-                pagerdutyIntegrationKey:
-                  title: PagerDuty integration key
-                  description: This value is ignored if you selected "Yes" for the previous question.
-                  type: string
+    - $yaml: ../defaults/parameters/pagerduty.yaml
```

Comment:
lmao this is the exact syntax extension I wrote half a year into my professional career. I think I used `!yaml`.

in any case, we should be *extremely* careful about what backstage ingests, because this is an ABC vuln waiting to happen

Reply:
ABC? Attack, Breach, Compromise?

Reply:
ArBitrary Codeexecution ? 🤔